pages/resources/faq.md

```diff
@@ -1,38 +1,71 @@
-The Portal Network aims to solve this problem by designing our networks so that each additional client that joints the network adds additional capacity to the network. The end result should be a network which becomes more robust and powerful as more nodes join the network.
 
 ## Why aren't there financial incentives?
 
 TLDR: Incorrectly implemented incentives lead to perverse incentives.
 
-Using financial incentives is a natural choice for many crypto/blockchain protocols. Well-designed incentives can be a powerful tool to encourage proper use and discourage malicious behaviour. Incentives might seem especially well-suited for the Portal Network, where the strength of the network is correlated to the number of participants and the amount of contributed computing resources. However, at the current point in time, Portal Network developers have opted out of implementing financial incentives.
+Using financial incentives such as tokens is common for many crypto/blockchain protocols. Well-designed incentives can be a powerful tool to encourage proper use and discourage malicious behaviour. Incentives might seem especially well-suited for the Portal Network, where the strength of the network is correlated to the number of participants and the amount of contributed computing resources. However, at the current point in time, Portal Network developers have opted out of implementing any form of financial incentives.
 
-The problem with financial incentives is they are complex and difficult to implement correctly. They also provide mechanisms for people to game the system and extract value rather than contribute towards overall network health. The Portal Network relies on producing a useful, lightweight client for developers, along with altruism and laziness, to encourage network adoption and sufficient computing resources.
-"Never underestimate the power of laziness and software defaults" - Piper
-Protocols like BitTorrent and IPFS are proof that this is a feasible strategy. Furthermore, protocols like Filecoin and The Graph Network already exist that use financial incentives to accomplish similar goals to the Portal Network.
+The problem with financial incentives is they are complex and difficult to implement correctly. They also provide mechanisms for people to game the system and extract value rather than contribute towards overall network health. The Portal Network aims to grow adoption by being useful and throuhg community altruism. We believe people will want to use Portal protocols because they enable much more convenient access to Ethereum and will naturally become critical Ethereum infrastructure. Protocols like BitTorrent and IPFS are proof that this is a feasible strategy.
```

Comment:
`throuhg` -> `through`

Reply:
```suggestion
The problem with financial incentives is they are complex and difficult to implement correctly. They also provide mechanisms for people to game the system and extract value rather than contribute towards overall network health. The Portal Network aims to grow adoption by being useful and through community altruism. We believe people will want to use Portal protocols because they enable much more convenient access to Ethereum and will naturally become critical Ethereum infrastructure. Protocols like BitTorrent and IPFS are proof that this is a feasible strategy.
```